File: Java/Car.java
package Java;

class Car {

    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver =  driver;
    }

    public void printDataCar() {
        System.out.println("Driver: " + driver.name + "\n" +
                           "License: " + license + "\n" +
                           "Passengers: " + passenger + "\n");
    }
    
}

File: PHP/Car.php
<?php

require_once('Account.php');
class Car {

    public $id;
    public $licence;
    public $driver;
    public $passenger;

    public function __construct($licence, $driver, $passenger)
    {
        $this->license = $licence;
        $this->driver = $driver;
        $this->passenger = $passenger;
    }

    public function printDataCard()
    {
        echo("Licencia: $this->license 
              <br> Driver: " . $this->driver->name .
              "<br> Document: " . $this->driver->document .
              "<br> Passengers: " . $this->passenger .
              "<br>"
            );
    }
}
